Core/clim-core/presentation-drag-and-drop.lisp

```diff
@@ -249,4 +249,5 @@
                    :window window
                    :x x
                    :y y
-                   :stream stream))))))
+                   :stream stream)))
+      (force-output stream))))
```

Comment:
shouldn't that be finish-output? are we OK with losing data if it can't be flushed immedietely?

Reply:
I'm not sure what losing data means in this context. Could you elaborate?

Reply:
`force-output` is specified as follows
```
 force-output initiates the emptying of any internal buffers but does not wait for completion or acknowledgment to return. 
```

if it is not possible to empty internal buffers immediately, then data may not be sent. semantically to ensure that output reached its destination you should use finish-output, which is specified as follows
```
 finish-output attempts to ensure that any buffered output sent to output-stream has reached its destination, and then returns. 
```

Or did I miss the purpose of this call entirely?

Reply:
The change is intended to fix the problem that pointer documentation is sometimes not displayed.

I think the problem is that the CLIM stream buffers until the line is complete or the output is explicitly forced/finished.

Regarding the difference between force and finish: I think force is appropriate because I want make sure the string is displayed at some point, but I don't want block until it is displayed on the screen (which implies an X protocol roundtrip for the `clx` backend) .

Reply:
makes sense, thanks.